### ISSUE TITLE
Games that don't need certain folders such as syswow64 or system32 will not start if those folders don't exist

### ISSUE THREAD

**Issue by McFranko** [ValveSoftware/Proton#4252]
I am using Proton 5.0-9
I saw this issue with Among Us
System Info
```
OS: Artix Linux
KERNEL: 5.8.14-artix1-1
CPU: Intel Core i7-8550U @ 1.80GHz
GPU: Intel Mesa Intel UHD 620 (KBL GT2)
GPU DRIVER: 4.6 Mesa 20.2.0
RAM: 16 GB
```

I've been trying to get Among Us to work for a while and I finally figured it out. I ran steam from a terminal and I could see the issue now. It gives this error:
`IsADirectoryError: [Errno 21] Is a directory: '/home/fronkles/Data/SteamLibrary/steamapps/compatdata/945360/pfx//drive_c/windows/syswow64/'`

I just went into the folder and ran `mkdir syswow64` which then caused this error to show up:

`IsADirectoryError: [Errno 21] Is a directory: '/home/fronkles/Data/SteamLibrary/steamapps/compatdata/945360/pfx//drive_c/windows/system32/'`

I did the same thing, just made that folder. After that, Among Us started working.

This sort of thing doesn't seem like it should produce an unrecoverable error. This seems like it should just make the directories and give you a warning in the logs about, so incase the program does need these folders you have a way to debug it. Or, these folders should just exist to begin with even if they don't have anything in them.

If this is a stupid complaint, lmk, I just thought it was worth mentioning

**Comment by kisak-valve** [comment]
Hello @McFranko, it'd be more interesting to figure out what happened that lead up to these folders missing. They should have been created and filled as part of the wine prefix setup on the first run of the game with Proton.

**Comment by McFranko** [comment]
I just downloaded steam from the arch repositories, and then linked up my game folder on my external drive and pressed play. I then tried to use the various version of proton (3.x to 5.x) and none of them worked. I tried reinstalling as well, that did nothing. I just checked and after I made those two empty directories, it look like they've been automatically populated. This a weird bug, it seems like it should have just made those directories and then let them get populated automatically
